com/truckmuncher/api/user.proto

```diff
@@ -0,0 +1,136 @@
+package com.truckmuncher.api.user;
+
+service UserService {
+
+    /**
+     * Link another social media account to your existing account. This can be used by regular
+     * users or vendors.
+     *
+     * This call requires authorization
+     */
+    rpc linkAccount (LinkAccountRequest) returns (User);
```

Comment:
wasn't sure how this was going to work, but the API code clears it up. almost creepy how we can do that without it seeming like we need to send data :P